src/components/Auth/AuthFooter.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC, memo } from 'react';
+import { Link } from 'react-router-dom';
+
+import { useCurrentCriteriaContext } from 'components/CurrentCriteriaProvider';
+import mitre from 'mitre.png';
+import mcode from 'mcode.png';
+import useStyles from './styles';
+
+const AuthFooter: FC = () => {
+  const styles = useStyles();
+  const { resetCurrentCriteria } = useCurrentCriteriaContext();
+
+  return (
+    <footer className={styles.footer}>
+      <Link to="/" className={styles.mitreButton} onClick={(): void => resetCurrentCriteria()}>
```

Comment:
Using a call back here (and in the `Link` below) instead of an arrow function will improve re-rendering the Link component